### PR TITLE
Migrate KebabOption to the SDK

### PR DESCRIPTION
**Pull request by fhlavac**
Migrate KebabOption component to the SDK as a part of ListPage migration effort
https://issues.redhat.com/browse/HAC-311

@andrewballantyne @invincibleJai

### CODE REVIEW

File: frontend/public/components/utils/kebab.tsx [hunk 1]
```diff
@@ -39,7 +38,9 @@ import {
   DeploymentModel,
   VolumeSnapshotModel,
 } from '../../models';
-import { ImpersonateKind } from '@console/dynamic-plugin-sdk';
+import { KebabOption, ImpersonateKind } from '@console/dynamic-plugin-sdk';
+
+export { KebabOption } from '@console/dynamic-plugin-sdk';
```
Comment (invincibleJai) [inline frontend/public/components/utils/kebab.tsx:43]: Duplicate imports `KebabOption`

File: frontend/packages/console-dynamic-plugin-sdk/src/app/components/utils/kebab.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
```
Comment (invincibleJai) [inline frontend/packages/console-dynamic-plugin-sdk/src/app/components/utils/kebab.tsx:1]: do we need this?

**Review by invincibleJai — CHANGES_REQUESTED** [review]
this PR just migrated one type `KebabOption`, this doesn't look like a big change could have been incorporated into original PR as well.

File: frontend/packages/console-dynamic-plugin-sdk/src/app/components/utils/kebab.tsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+import { AccessReviewResourceAttributes } from '../../../extensions/console-types';
+
+export type KebabOption = {
```
Comment (invincibleJai) [inline frontend/packages/console-dynamic-plugin-sdk/src/app/components/utils/kebab.tsx:4]: if this is just one type, then we don't need to have `kebab.tsx` here, we do have `console-types.ts` and `common-types.ts`

File: frontend/public/components/utils/kebab.tsx (hunk 1 above)
Comment (invincibleJai) [reply frontend/public/components/utils/kebab.tsx:43]: ah it's export, now issues

**Comment by fhlavac** [comment]
Closing this and moving the changes to https://github.com/openshift/console/pull/10515

**Pull request closed without merging**